File: Sobrecarga/java/Seconds.java
public class Seconds{
    // Atributos
    public float seg;
    public int min;
    public int hrs;
    public int dia;

    // Constructor 1
    public Seconds(int min){
        this.min = min;
        this.hrs = 0;
        this.dia = 0;
        this.seg = 0;
    }

    // Constructor 2
    public Seconds(int hrs, int min){
        this.min = min;
        this.hrs = hrs;
        this.dia = 0;
        this.seg = 0;
    }

    // Constructor 3
    public Seconds(int dia, int hrs, int min){
        this.min = min;
        this.hrs = hrs;
        this.dia = dia;
        this.seg = 0;
    }

    // Metodos
    public void convert(){
        this.seg = this.min*60;
        this.seg += this.hrs*3600;
        this.seg += this.dia*3600*24;
    }
}

File: Sobrecarga/cpp/Seconds.cpp
#include "Seconds.h"

Seconds::Seconds(int m){
    min = m;
    hrs = 0;
    dia = 0;
    seg = 0.0;
}

Seconds::Seconds(int h, int m){
    min = m;
    hrs = h;
    dia = 0;
    seg = 0.0;
}

Seconds::Seconds(int d, int h, int m){
    min = m;
    hrs = h;
    dia = d;
    seg = 0.0;
}

Seconds::~Seconds(){};

void Seconds::convert(){
    seg = min*60;
    seg += hrs*3600;
    seg += dia*3600*24;
}